drift/lib/src/runtime/manager/filter.dart

```diff
@@ -300,29 +307,29 @@ class ComposableFilter extends _Composable {
   Expression<bool>? expression;
 
   /// Create a new [ComposableFilter] for a column with joins
-  ComposableFilter._(this.expression, this.joinBuilders);
+  ComposableFilter.$_(this.expression, this.joinBuilders);
```

Comment:
Is the rename because this is now meant to be called from generated code directly? In that case we should expand the doc comment to mention that this is not meant to be called by user code (or maybe just mark it as `@internal` since generated code ignores lints).

Reply:
Turns out, that this should've been reverted along with https://github.com/simolus3/drift/pull/3062/commits/63ba8fec47ac6915ac1fff87096464f2cc1b920c .
I'll reverse this too.